academy/3-my-own-chain/migration.md

```diff
@@ -990,19 +990,19 @@ The processing **routines** will be divided as per the following:
 1. The **game processing** routine will:
     
     * Receive separate arrays of games from the _stored-game_ channel.
-    * Compute the aggregate player infos from them. I.e. **_map_**.
+    * Compute the aggregate player info files from them. I.e. **_map_**.
```

Comment:
Mmh, no _files_ does not work well here. All is happening in memory and files are understood as on disk.

_Records_ or _objects_ works better.